app/component/itinerary/ItineraryListHeader.js

```diff
@@ -31,6 +32,25 @@ export default function ItineraryListHeader({
           </div>
         </div>
       )}
+      {showCarBoardingInfo && (
+        <div className="with-car-info">
+          <div className="with-car-icon notification-icon">
+            <ServiceAlertIcon
+              className="inline-icon"
+              color="#007AC9"
+              severityLevel={AlertSeverityLevelType.Info}
+            />
+          </div>
+          <div className="with-car-info-notification">
```

Comment:
merge these 2 divs: with-car-info-notification and with-car-info-content

Reply:
I removed with-bike-info-header and with-car-info-header in itinerary-list-header.scss, they were not used anymore. I merged with-bike-info-content and with-bike-info-notification as well as with-car-info-content and with-car-info-notification.

Reply:
I noticed a new css class  .with-car-info-notification-icon which seems to be unused.

Reply:
I removed with-car-info-notification-icon and with-bike-info-notification-icon. The car class was accidentally created based off of the old bike icon class. Both seem unused and I removed them